amazon-kinesis-client/src/test/java/software/amazon/kinesis/lifecycle/ShutdownTaskTest.java

```diff
@@ -77,7 +78,7 @@
     private static final ShutdownReason LEASE_LOST_SHUTDOWN_REASON  = ShutdownReason.LEASE_LOST;
     private static final MetricsFactory NULL_METRICS_FACTORY = new NullMetricsFactory();
 
-    private final String concurrencyToken = "testToken4398";
+    private final String concurrencyToken = "0-1-2-3-4";
```

Comment:
Why did we have to change this?

Reply:
Because the old format would fail the format check in UUID.fromString method